c_src/dpi_nif.c

```diff
@@ -0,0 +1,85 @@
+/*
+20181121 IMPLEMENT nif with eunit test for
+* dpiContext_*
+* dpiConn_*
+* dpiStmt_*
+*/
+#ifdef EMBED
+#include "odpi/embed/dpi.c"
+#endif
+
+#include "dpi_nif.h"
+#include "stdio.h"
+#include "dpiContext_nif.h"
+#include "dpiConn_nif.h"
+#include "dpiStmt_nif.h"
+#include "dpiQueryInfo_nif.h"
+
+ERL_NIF_TERM ATOM_OK;
+ERL_NIF_TERM ATOM_NULL;
+ERL_NIF_TERM ATOM_ERROR;
+ERL_NIF_TERM ATOM_ENOMEM;
+ERL_NIF_TERM ATOM_UNIMPLIMENTED;
+ERL_NIF_TERM TUPLE_ERROR_UNIMPLIMENTED;
```

Comment:
`TUPLE_ERROR_UNIMPLIMENTED` not needed as, `{error, unimplimented}` term is created at return in `UNIMPLIMENTED`  macro